pages/workspaces/[workspaceId]/activity.tsx

```diff
@@ -63,32 +54,36 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   }
 
   setCookie({ response: context.res, name: WORKSPACE_ID_COOKIE_NAME, value: workspaceId });
-  const featureFlags = await getAllFeatureFlags(userId);
 
   return {
-    props: { workspace: data, overLimit: !!data?.exceeds_upgrade_limits, isOwner, bearerToken, userId, featureFlags },
+    props: {
+      workspace: data,
+      overLimit: !!data?.exceeds_upgrade_limits,
+    },
   };
 };
 
 interface WorkspaceDashboardProps {
-  userId: number;
   workspace: Workspace;
-  isOwner: boolean;
   overLimit: boolean;
-  bearerToken: string;
-  featureFlags: Record<FeatureFlag, boolean>;
 }
 
 type OrderDirection = "ASC" | "DESC";
 
-const WorkspaceActivityPage = ({
-  workspace,
-  isOwner,
-  overLimit,
-  bearerToken,
-  userId,
-  featureFlags,
-}: WorkspaceDashboardProps) => {
+const WorkspaceActivityPage = ({ workspace, overLimit }: WorkspaceDashboardProps) => {
+  const { sessionToken, signIn, userId } = useSupabaseAuth();
```

Comment:
This page, the activity pages (pull requests, issues) load the user info client-side now because when you log in if you were logged out, the page does not rerun getServerSideProps and rerender server-side, so we need to handle this client-side. That would probably change if/when we move to Next.js's app router.